draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4030,6 +4030,13 @@ bytes=500-700,601-999
    match multiple media types that differ only in processing semantics.
    Implementers are encouraged to provide a means to disable such sniffing.
 </t>
+<t>
+   Furthermore, although Content-Type is defined as a singleton header, it is
+   sometimes sent as a list (i.e., with multiple, comma-separated values).
+   Recipients &MAY; attempt to handle this error by using the last syntactically
+   valid member of the list. Note that some implementations might have additional
+   error handling behaviors; e.g., see <xref target="FETCH"/>.
+</t>
```

Comment:
Does this imply that they MUST NOT handle the error differently? In which case I would object to this as it is a normative change. If it's not, it shouldn't use "MAY".

Reply:
I also don't think it is good advice. I've never seen an application choose the last value on error.

Reply:
MAY changed to can.

@royfielding see issue discussion; this is how all browsers handle it (with [tests](https://wpt.fyi/results/fetch/content-type/response.window.html?label=experimental&label=master&aligned)), so it seems worth mentioning as an option (but not mandating).